client/src/components/NoteTags.js

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+
+export default function NoteTags({ onCreateTag, tags, setTags, onDeleteTag }) {
+  const [tagValue, setTagValue] = useState('');
+
+  function handleChange(event) {
+    setTagValue(event.target.value);
+  }
+
+  function handleKeyDown(event) {
+    if (event.key === 'Enter') {
+      event.preventDefault();
+      const memberTags = [...tags, tagValue];
+      setTags(memberTags);
+      onCreateTag(tagValue);
+      setTagValue([]);
```

Comment:
```suggestion
      setTagValue('');
```
Why an array, the initial state is an empty string?

Reply:
Ohh yes, thanks for checking!